tests/rptest/tests/rpk_topic_test.py

```diff
@@ -318,16 +318,21 @@ def test_analyze(self):
         assert topic in res
 
         def within(actual, expected, err) -> bool:
-            return (actual >= expected * (1 - err)) & (actual <= expected *
-                                                       (1 + err))
+            res = (actual >= expected * (1 - err)) & (actual <= expected *
```

Comment:
`&` is bitwise-and in Python, should be `and`

could this be causing the flakiness?

Reply:
I guess not since looks boolean on both sides but anyway this is wrong (though it probably has no effect)